plugins/modules/api.py

```diff
@@ -73,118 +136,76 @@
 
 EXAMPLES = '''
 ---
-- name: Use RouterOS API
-  hosts: localhost
-  gather_facts: no
-  vars:
-    hostname: "ros_api_hostname/ip"
-    username: "admin"
-    password: "secret_password"
-
-    path: "ip address"
-
-    nic: "ether2"
-    ip1: "1.1.1.1/32"
-    ip2: "2.2.2.2/32"
-    ip3: "3.3.3.3/32"
-
   tasks:
```

Comment:
Why not remove this one as well, and remove the four leading spaces from all tasks?